ansible-ipi-install/inventory/hosts

```diff
@@ -0,0 +1,11 @@
+[masters]
```

Comment:
people seem to prefer yaml inventory instead of ini format those days

Reply:
Since we are talking preference here, does it matter?